### ISSUE TITLE
Add 180-degree turn to driver controlled

### ISSUE THREAD

**Issue by pdxaeroman** [LaSalleRobots/PowerPlay#4]
So like this?

**Comment by lukasmwerner** [comment]
Will this be a one button turn or what is the goal here?

**Comment by pdxaeroman** [comment]
yes, much like the presets that the lift operator gets. this was cozzoli's idea. the driver presses a button and the robot flips around at maximum speed.

**Comment by lukasmwerner** [comment]
Nice! ok so we should probably use the IMU's reading to get this. Do you want to own this? @pdxaeroman

**Comment by pdxaeroman** [comment]
yeah, should be easy enough to implement once we have a reliable way of interpreting IMU

**Comment by pdxaeroman** [comment]
I built... something. Reopen this if a rebuild is necessary.

**Comment by lukasmwerner** [comment]
This is now sorta done. But not based on the IMU itself

**Comment by pdxaeroman** [comment]
Needs to work better and at higher speeds
